app/src/main/res/values/strings.xml

```diff
@@ -3,4 +3,8 @@
     <string name="title_main">Parking System</string>
     <string name="text_main_activity_parking_size_button_label">Configure parking size</string>
     <string name="text_parking_size_toast">Total of parking lots: %1$d</string>
+    <string name="toast_parking_dialog_error_input">Please, enter a valid value</string>
+    <string name="text_parking_size_dialog">Please enter a number between 1 and %1$d</string>
+    <string name="ok_button">OK</string>
+    <string name="cancel_button">CANCEL</string>
```

Comment:
rename this resources name

Reply:
Done

Reply:
I change like "button_parking_size_dialog_ok" and "button_parking_size_dialog_cancel".
Like the code style pattern from mobile: <WHAT>_<WHERE>_<DESCRIPTION>

Reply:
Done